h5py/tests/test_group.py

```diff
@@ -51,19 +51,6 @@ def tearDown(self):
         if self.f:
             self.f.close()
 
-class TestRepr(BaseGroup):
```

Comment:
This class was replaced by another `TestRepr` below in the same file, so it wasn't getting run.